direction/index.md

```diff
@@ -1,42 +1,60 @@
+We will measure this by:
 
-## In 3-6 months, Sourcegraph will have:
+- Number of daily active users
+- Number of times Sourcegraph is used per day
 
-### Improved usability
+#### How we're going to do this
 
-Sourcegraph is intuitive to use for a wide range of roles, from developers to PMs, engineering managers, data analysts, and more. It adds value to this wide range of roles by making more information about code available, such as language statistics and dependency graphs. [Search has an improved UI](https://about.sourcegraph.com/blog/sourcegraph-3.10#improved-autocompletion-for-search-query-filters) that makes it more accessible, easier to use, and faster to drill down on what you're looking for.
+- Insert Sourcegraph into more places in the developer workflow, so developers are being prompted to go use Sourcegraph.
+- Give developers more reasons to come back with greater frequency by adding more value for them inside of Sourcegraph.
 
-Improving upon on our [basic code intelligence](https://docs.sourcegraph.com/user/code_intelligence) that works for every language, Sourcegraph provides precise code intelligence for a subset of common languages including Go, TypeScript, C/C++, Java, and C#.
+### Support complex configurations and workflows
 
-### Enhanced scalablity, reliability, and security
+#### Challenge to overcome
 
-Sourcegraph performs at the scale of our largest customers under a wide variety of configurations and deployment infrastructure. [Code search is fast at large scale](https://docs.google.com/document/d/18w8T_KzYxQye8wg1g01QpMOX4_ERTtbOxMBRYaOEkmk/edit?usp=sharing) (~80k repositories), and the API can support enterprise level usage. Sourcegraph enforces repository permissions using ACLs from Bitbucket Server, GitHub, and GitLab. Admins understand the health of their instances and are alerted proactively if things fail.
+Many organizations have unique code setups and workflows that require workarounds to support. We want Sourcegraph to be the code search solution for every developer and every organization, regardless of their setup.
 
-### Code change management
+#### What we want to achieve
 
-Use Sourcegraph to [run large-scale code change campaigns](https://about.sourcegraph.com/product/code-change-management) to remove legacy code, fix critical security issues, and pay down tech debt. You can create campaigns across thousands of repositories and code owners. Sourcegraph automatically creates and updates all of the branches and pull requests, and you can track progress and activity in one place.
+At any organization, Sourcegraph is how developers are able to work with large and complex development workflows. Sourcegraph sustainably supports configuration of unique infrastructure and code setups.
+
+#### How we're going to do this
+
+- Solve use cases that are present at many large enterprises.
+- Reduce friction in getting Sourcegraph deployed at scale.
+- More integrated support for non-Git version control systems.
```

Comment:
Is this true? I can't recall where exactly but I remember I saw on Slack that Git is the only version control system we would put effort on.

Just pointing out, and I have no problem with this statement.

Reply:
Good question! Over the past quarter we have gotten very strong signal from customers and prospects that supporting non-Git is important. We want Sourcegraph to be how _every_ developer finds code, not just developers using Git. 

We don't need to make every other VCS as good as the support we have for Git (at least not this year), but it does need to be far better than it is now. It should feel like an integrated experience of using Sourcegraph, not a workaround or bandaid.